common/server-sent-events.conf

```diff
@@ -4,5 +4,4 @@ proxy_cache off;
 # make event source work with nginx
 # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
 proxy_set_header Connection '';
-proxy_http_version 1.1;
```

Comment:
Because HTTP/1.1 isn't required for SSE as far as I know, and isn't a requirement of logging which uses it. @darkonie can you correct me if I'm wrong here?